client/src/components/pageBuilder/ComponentRegistry.js

```
@@ -0,0 +1,161 @@
+			'product.imageUrl': { type: 'text', label: 'Image URL' },
+			'product.stock': { type: 'number', label: 'Stock' }
+		}
+	},
+	'text-block': {
+		id: 'text-block',
+		name: 'Text Block',
+		category: 'Content',
+		component: ({ content, variant = 'body1' }) => React.createElement('div', {
+			style: { margin: '16px 0' }
+		}, React.createElement('div', {
+			dangerouslySetInnerHTML: { __html: content || 'Add your text here...' }
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)